router/router.go

```diff
@@ -64,6 +64,11 @@ type HandleDestOAuthRespParamsT struct {
 	secret         json.RawMessage
 }
 
+type readProcessStruct struct {
+	combinedList  []*jobsdb.JobT
+	latenciesUsed map[string]float64
+}
+
```

Comment:
please avoid using the type in the name, e.g. I would call it 
```suggestion
type jobsWithLatencies struct {
	jobs  []*jobsdb.JobT
	latencies map[string]float64
}

```